TGM/src/main/java/com/minehut/tgm/modules/DeathModule.java

```diff
@@ -18,14 +16,16 @@
      * This is to prevent players from "combat logging"
      * by team swapping to avoid getting a death.
      */
-    @EventHandler
-    public void onTeamChange(TeamChangeEvent event) {
-        if (TGM.get().getMatchManager().getMatch().getMatchStatus() == MatchStatus.MID) {
-            if (event.getOldTeam() != null && !event.getOldTeam().isSpectator()) {
-                event.getPlayerContext().getPlayer().setHealth(0);
-            }
-        }
-    }
+
+    // not need because i prevent people changing teams during the game
+//    @EventHandler
+//    public void onTeamChange(TeamChangeEvent event) {
+//        if (TGM.get().getMatchManager().getMatch().getMatchStatus() == MatchStatus.MID) {
+//            if (event.getOldTeam() != null && !event.getOldTeam().isSpectator()) {
+//                event.getPlayerContext().getPlayer().setHealth(0);
+//            }
+//        }
+//    }
```

Comment:
I haven't reviewed the code, I don't intend to take any kind of action on the PR. I just was getting e-mails about your comments and found an issue, so I pointed it out. Thank you for explaining.